docs/api.rst

```diff
@@ -74,6 +74,19 @@ The ``pika/init.hpp`` header provides functionality to manage the pika runtime.
 ======================
 
 The ``pika/execution.hpp`` header provides functionality related to ``std::execution``.
+``std::execution`` functionality is defined in the ``pika::execution::experimental`` namespace. When
```

Comment:
It might be worth making it explicit here that the same namespace also provides additional functionality not provided by the standard?

Reply:
Good idea. Do you think this is explicit enough: https://github.com/pika-org/pika/compare/5ca735f90ff3bb88c51c486fcd35230ad49df756..5d47c715712df0bdf522b855844681ac68f3813d?

Reply:
Yes, thanks.